rwengine/src/loaders/LoaderCOL.cpp

```diff
@@ -48,19 +49,19 @@ bool LoaderCOL::load(const std::string& path) {
         model->modelid = head.modelid;
 
         auto readFloat = [&]() {
-            auto f = (float*)d;
+            auto f = bit_cast<float>(*d);
```

Comment:
nit: `d` is `auto`, so if it's suddenly not `sizeof(float)` it *looks* like you've changed semantics. This is actually poor design in the `bit_cast` function in my opinion.

Reply:
I'll add better docs.